src/components/signUp/signup.tsx

```diff
@@ -44,8 +47,10 @@ const signUp = () => {
 		} else {
 			await signUp({ email, password })
 				.unwrap()
-				.then(() => {
+				.then(res => {
 					setSucess(true);
+					dispatch(saveUserId(res.id));
```

Comment:
save user by id? if yes then its a bad option, we have a token to understand what user is it, so we don't need to path parameter

Reply:
i need this id to send post request